build.sbt

```diff
@@ -34,8 +34,12 @@ lazy val higherKindCore = project settings (
   publishName := "core-higher-kind",
   libraryDependencies ++= Seq(catsCore, catsFree, catsTagless),
 )
-
-lazy val core = project dependsOn (opticsCore, higherKindCore) settings (
+lazy val kernel = project in file("kernel") dependsOn (opticsCore, higherKindCore) settings (
+  defaultSettings,
+  publishName := "kernel",
+  libraryDependencies ++= Seq(catsCore, catsTagless),
+)
+lazy val core = project dependsOn (opticsCore, higherKindCore, kernel) settings (
```

Comment:
Do we need to depend on `opticsCore` and `higherKindCore` since `kernel` already depends on them?